docs/getting-started/email-setup.md

```diff
@@ -0,0 +1,12 @@
+# Email Setup
+
+Logging emails is a central activity to Hubspot and allows for greater transparency and provides an opportunity for team members to have a deeper understanding of where a relationship is at. When configured Hubspot email logging allows to set up automated and/or selective logging of emails while you are in your prefered email tool
```

Comment:
> Logging emails is a central activity to Hubspot and allows for greater transparency and provides an opportunity for team members to have a deeper understanding of where a relationship is at.

Edit: Logging emails is central to Hubspot’s ability to track interactions and allows for greater transparency, providing an opportunity for team members to have a deeper understanding of where a relationship is at.

> When configured Hubspot email logging allows to set up automated and/or selective logging of emails while you are in your prefered email tool

Edit: **Once** configured**,** Hubspot....prefer**r**ed email tool**.**

Reply:
@laneymangan For this edit:
>Edit: Once configured**,** Hubspot....preferred email tool**.**

Did you want me to add 4 Astrix or make that section bold?

Reply:
@AnthonyRollins Sorry this one looks crazy lol. Edits are supposed to be: add a comma after configured, add a period after tool and add an r to "prefered". And (definitely optional) change "when" to "once"

Reply:
Got it. I've updated that content.